samples/subsys/smf/psicc2/prj.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_LOG=y
+CONFIG_SHELL=y
+
+# Needed for boards that enable RTT backends for logging
+# e.g. nrf52840dk/nrf52840
+CONFIG_LOG_BACKEND_RTT=n
```

Comment:
This should probably go into `boards/nrf52840dk/nrf52840.conf`

Reply:
I feel it would be best to leave it here, to prevent surprises on other boards which may use `CONFIG_LOG_BACKEND_RTT`. I've not made a survey, and if someone was to add a board later I wouldn't know about it.